client/src/components/Aside.js

```diff
@@ -0,0 +1,14 @@
+import styled from "@emotion/styled";
+
+const Aside = styled.div`
```

Comment:
Aside doesnt tell me really, what it does

Reply:
Thx! It's an styling issue, I have already a semantic footer component